dashboard/app/views/admin_reports/admin_stats.html.haml

```diff
@@ -65,27 +55,28 @@
         %li
           30-Day Male Students:
           = @stats['actives']['thirty_day_male_students']
-    - ["historical", "current_weekly", "current_monthly"].each do |type|
+    - {'historical': 'accounts at least 37 days old and logged in at least once after 30 days',
+       'current_weekly': 'accounts 37 to 44 days old and logged in at least once after 30 days',
+       'current_monthly': 'accounts 60 to 90 days old and logged in at least once after 30 days'}.each do |type, description|
       %li
         Retention
-        = type
-        (Percentages)
+        = "#{type} (#{description})"
         %ul
           %li
             30-Day Student:
-            = @stats['thirty_day_retention_' + type]['students']
+            = (100.0 * @stats["thirty_day_retention_#{type}"]['students']).to_s + '%'
```

Comment:
this section could presumably also be structured as a for loop over something like `{'students': '30-Day Student', ...}`